src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -116,34 +148,69 @@ void EnHoll_SwapRooms(PlayState* play) {
     play->roomCtx.unk_30 ^= 1;
 }
 
-// Horizontal Planes
-void func_80A58DD4(EnHoll* this, PlayState* play) {
+/**
+ * These are all absolute distances in the relative z direction. That is, moving
+ *   towards or away from the "face" of the holl regardless of orientation.
+ * Moving within these distances of the holl have the following effects:
+ * [0] : Load the room on this side of the holl if not already loaded
+ * [1] : Load the room on the other side of the holl
+ * [2] : Start of fade region, where the plane is fully opaque
+ * [3] : End of fade region region, where the plane is fully transparent
+ *
+ * Within the fade region, the plane goes:
+ *   opaque -> transparent if approaching,
+ *   transparent -> opaque if receding
+ */
+static f32 sHorizontalVisibleNarrowTriggerDists[2][4] = {
+    { 200.0f, 150.0f, 100.0f, 50.0f }, // default
+    { 100.0f, 75.0f, 50.0f, 25.0f },   // SCENE_SPIRIT_TEMPLE
+};
```

Comment:
Considering how much documentation is being done on this one structure, it might make sense to just define a type for it right here, and then in-line the data within the function.

Reply:
by define a type do you mean use a struct? dont think the codegen would be the same

Reply:
Yes, a struct, and I've already tested that it OKs perfectly fine.

Reply:
Since this type is jank I don't know what to name the struct members though
Will be clearer when I understand the room loading system